sdks/python/test-suites/dataflow/common.gradle

```diff
@@ -124,6 +124,30 @@ task postCommitIT {
   }
 }
 
+task postCommitSickbay {
+  dependsOn 'installGcpTest'
+  dependsOn ':sdks:python:sdist'
+  dependsOn ':runners:google-cloud-dataflow-java:worker:shadowJar'
+
+  def dataflowWorkerJar = project(":runners:google-cloud-dataflow-java:worker").shadowJar.archivePath
```

Comment:
I don't think we need this. I'd like to run these on Runner V2 only. Do you know how to set this up as a Runner V2 test?

Reply:
I don't know how to set up as a Runner V2.
I did in this way to be sure that the sickbay test have the same configuration as normal post commit tests

Reply:
You would be `--data_service_options=use_runner_v2` I believe. It may need to be `--experiments=use_runner_v2` if that does not work (but it is not an experiment any more).

Pablo - is `apache-beam-testing` excluded from auto opt in?